src/connectors/current-refinements/connectCurrentRefinements.ts

```diff
@@ -316,6 +316,7 @@ function clearRefinementFromState(
   state: SearchParameters,
   refinement: CurrentRefinementsConnectorParamsRefinement
 ): SearchParameters {
+  state = state.resetPage();
```

Comment:
doesn't each individual method already reset the page? eg. [removeDisjunctiveFacetRefinement](https://community.algolia.com/algoliasearch-helper-js/reference.html#AlgoliaSearchHelper#removeDisjunctiveFacetRefinement)

Reply:
It does when [called on the helper](https://github.com/algolia/algoliasearch-helper-js/blob/d9ebb01382ec83e0c579c393c3e9df83a9261699/src/algoliasearch.helper.js#L658-L665) but [does not on a `SearchParameters` instance](https://github.com/algolia/algoliasearch-helper-js/blob/fc183ec2910d4ba33067b3487f3a70e940fd24d3/src/SearchParameters/index.js#L846-L856) 🤔

Reply:
And I guess we do it with `SearchParameters` because `clearRefinementFromState` is also used for `createURL` ?

Reply:
ah yes of course, makes sense!